src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,21 @@
+@import './utilits/variables';
+@import './utilits/media';
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: $space-between-cards;
+  justify-content: center;
+  max-width: $catalog-max-width;
+  margin: 0 auto;
```

Comment:
I don't think that block positioning is good idea

Reply:
I am agree make this block someone's element & set this margins there

Reply:
Done.